### PR TITLE
new next 12 month can not hearing screen

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-4676

### Change description ###
'Are there any dates in the next 12 months when you, your experts or witnesses cannot go to a hearing' screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/en.json [hunk 1]
```diff
@@ -1046,6 +1046,11 @@
       "PAGE_TITLE": "New response deadline",
       "TITLE": "4 pm on {{ responseDeadline }}"
     },
+    "NEXT_12MONTHS_CANNOT_HEARING": {
```
Comment (portizdev) [inline src/main/modules/i18n/locales/en.json:1049]: You forgot to put it in cy.json as well.

File: src/main/common/form/models/directionQuestionnaireErrorMessages.ts [hunk 2]
```diff
@@ -4,4 +4,5 @@ export const DirectionQuestionnaireErrorMessages = {
   requestExtra4weeks: 'ERRORS.VALID_REQUEST_EXTRA_4_WEEKS',
   sharedExpert: 'ERRORS.VALID_SHARED_EXPERT',
   triedToSettle: 'ERRORS.VALID_TRIED_TO_SETTLE',
+  next12MonthsCanNotHearing: 'ERRORS.VALID_NEXT_12MONTHS_CANNOT_HEARING',
```
Comment (commitBlob) [inline src/main/common/form/models/directionQuestionnaireErrorMessages.ts:7]: this variable name is so hard to read
maybe renaming it into cantAttendHearingInNext12Months

File: src/main/modules/i18n/locales/en.json (hunk 1 above)
Comment (commitBlob) [inline src/main/modules/i18n/locales/en.json:1049]: did you forget to include welsh keys?

File: src/main/routes/urls.ts [hunk 3]
```diff
@@ -142,3 +142,4 @@ export const DQ_OTHER_WITNESSES_AVAILABILITY_DATES_FOR_HEARING_URL = `${DIRECTIO
 export const DQ_EXPERT_GUIDANCE_URL = `${DIRECTIONS_QUESTIONNAIRE_URL}/expert-guidance`;
 export const CLAIM_CLAIMANT_INDIVIDUAL_DETAILS_URL = `${BASE_CLAIM_URL}/claimant-individual-details`;
 export const CLAIMANT_DOB_URL = `${BASE_CLAIM_URL}/claimant-dob`;
+export const DQ_NEXT_12MONTHS_CAN_NOT_HEARING_URL = `${DIRECTIONS_QUESTIONNAIRE_URL}/next-12Months-can-not-hearing`;
```
Comment (TatsyMaslova) [inline src/main/routes/urls.ts:145]: and rename this one too

File: src/main/routes/features/directionsQuestionnaire/next12MonthsCanNotHearingController.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import * as express from 'express';
```
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/routes/features/directionsQuestionnaire/next12MonthsCanNotHearingController.ts:1]: I totally Agree

File: src/main/routes/urls.ts (hunk 3 above)
Comment (laarmada) [reply src/main/routes/urls.ts:145]: I totally Agree I will refactor in everywhere

File: src/main/common/form/models/directionQuestionnaireErrorMessages.ts (hunk 2 above)
Comment (laarmada) [reply src/main/common/form/models/directionQuestionnaireErrorMessages.ts:7]: I totally Agree I will refactor in everywhere

File: src/main/modules/i18n/locales/en.json (hunk 1 above)
Comment (laarmada) [reply src/main/modules/i18n/locales/en.json:1049]: :)

**Review by TatsyMaslova — APPROVED** [review]

**Pull request merged**